Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [requires]
-python_version = "3"
+python_version = "3.10.0"
```

Comment:
Don't do this, setting 3.10 in this file straight excludes earlier python versions.

Reply:
What would be a good version just "3" ... I can set it back. No preference here. As long as it works

Reply:
3  will make it not exclude people on older versions of python